codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithypython/localservice/extensions/DafnyPythonLocalServiceStructureGenerator.java

```diff
@@ -374,13 +374,6 @@ protected void writeFromDict(boolean isError) {
       () -> {
         writer.writeDocs(() -> {
           writer.write("Creates a $L from a dictionary.\n", shapeName);
-          writer.write(
```

Comment:
This is unrelated removal of comments that weren't supposed to be user-facing in the first place?

Reply:
This comment is inaccurate for Python codegen (and is inaccurate for Smithy-Python codegen).
* Formatting according to docstring note: `SomeMemberName`
  * This is the "boto3" formatting.
* Actual Smithy-Python formatting: `someMemberName`
* Smithy-Dafny Python formatting: `some_member_name`